### PR TITLE
Add integration test for Parser and disable skip_until_parameters? option for RemoteStream

### PR DESCRIPTION
**Pull request by Rados13**

### CODE REVIEW

File: lib/membrane_h264_ffmpeg/parser.ex [hunk 1]
```diff
@@ -252,7 +252,11 @@ defmodule Membrane.H264.FFmpeg.Parser do
       Enum.concat([[state.frame_prefix || <<>>], sps, pps])
       |> Enum.join(<<0, 0, 1>>)
 
-    {:ok, %{state | frame_prefix: frame_prefix}}
+    if state.skip_until_parameters? do
+      Membrane.Logger.warn("Flag skip_until_parameters? is not compatible with H264.RemoteStream")
```
Comment (daniel-jodlos) [inline lib/membrane_h264_ffmpeg/parser.ex:256]: I think we should be a little more verbose about the action that we are taking.
```suggestion
      Membrane.Logger.warn("""
      Flag skip_until_parameters? is not compatible with Membrane.H264.RemoteStream caps.
      It is being automatically disabled.
      """)
```

File: test/parser/parser_integration_test.exs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  @fixtures_dir "./test/fixtures/"
+  @stream_with_params File.read!("test/fixtures/input-10-720p.h264")
+  @input_caps %Membrane.H264.RemoteStream{
+    decoder_configuration_record:
+      <<1, 2, 131, 242, 255, 225, 0, 8, 103, 66, 0, 13, 233, 2, 131, 242, 1, 0, 5, 104, 206, 1,
+        15, 32>>,
+    stream_format: :byte_stream
+  }
+
+  defp chunk_bits(binary, n) do
```
Comment (daniel-jodlos) [inline test/parser/parser_integration_test.exs:17]: https://hexdocs.pm/bunch/Bunch.Binary.html#chunk_every/2

File: test/parser/parser_integration_test.exs [hunk 3]
```diff
@@ -0,0 +1,80 @@
+defmodule Membrane.H264.FFmpeg.Parser.IntegrationTest do
+  use ExUnit.Case
+  import Membrane.ParentSpec
+  import Membrane.Testing.Assertions
+
+  alias Membrane.Testing
+
+  @fixtures_dir "./test/fixtures/"
+  @stream_with_params File.read!("test/fixtures/input-10-720p.h264")
```
Comment (daniel-jodlos) [inline test/parser/parser_integration_test.exs:9]: I think we should be feeding the stream without the parameters in order to actually test anything

File: test/parser/parser_integration_test.exs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    reference_file = Path.join(@fixtures_dir, "reference.h264")
+
+    {:ok, pipeline} =
+      create_pipeline(input_file, output_file)
+      |> Testing.Pipeline.start_link()
+
+    play_and_validate(pipeline, reference_file, output_file)
+  end
+
+  defp create_pipeline(input_file, output_file) do
```
Comment (daniel-jodlos) [inline test/parser/parser_integration_test.exs:44]: I don't think you're using `input_file` parameter

**Review by daniel-jodlos — CHANGES_REQUESTED** [review]

**Comment by Rados13** [comment]
Solved in #60

**Pull request closed without merging**